jetty-core/jetty-compression/jetty-compression-common/src/main/java/org/eclipse/jetty/compression/Compression.java

```diff
@@ -82,18 +81,6 @@ public String etag(String etag)
         return etag + etagSuffix;
     }
 
-    public int getBufferSize()
```

Comment:
This is public API so we should not remove these methods in a dot release, these should be deprecated instead.